src/components/ScenarioParameters/components/ScenarioParametersInputs/components/TableDeleteRowsDialog/TableDeleteRowsDialog.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { DontAskAgainDialog } from '@cosmotech/ui';
+import PropTypes from 'prop-types';
+
+export const TableDeleteRowsDialog = (props) => {
+  const { t } = useTranslation();
+  const { open, onClose, onConfirm, SelectedRowsCount } = props;
+  const labels = {
+    title: t('genericcomponent.table.deleteRowsDialog', 'Delete selected lines?'),
+    body: t('genericcomponent.table.deleteRowsDialog', { count: SelectedRowsCount }),
+    cancel: t('genericcomponent.table.deleteRowsDialog', 'Cancel'),
+    confirm: t('genericcomponent.table.deleteRowsDialog', 'Delete'),
+    checkbox: t('genericcomponent.table.deleteRowsDialog', "Don't show this message again"),
```

Comment:
You're always using the same translation key `genericcomponent.table.deleteRowsDialog`, it probably doesn't work as expected :slightly_smiling_face: